src/utils/featch.wrapper.ts

```diff
@@ -0,0 +1,43 @@
+// utils/fetchWrapper.ts
+
+const fetchWrapper = async (
+  url: string,
+  method: string,
+  data?: object | null,
```

Comment:
Why is data's type null and undefined both? If possible, only either one of them should be used. Ignore if both are intentional.